### DISCUSSION TITLE
zingg-0.3.0-SNAPSHOT-spark-3.0.3

### DISCUSSION THREAD

**Discussion by sonalgoyal** [zinggAI/zingg#23] [Announcements]
Zingg 0.3.0 with Spark 3.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/zinggAI/zingg/releases/tag/'>zingg-0.3.0-SNAPSHOT-spark-3.0.3</a>.</em>

**Comment by jameselieta** [comment]
hello, @sonalgoyal I really appreciate the great work you are doing.

**Reply by sonalgoyal** [reply]
Thanks a lot @jameselieta ☺️🤩
